stratum/hal/bin/tdi/dpdk/dpdk_main.cc

```diff
@@ -30,10 +30,11 @@
 #include "stratum/lib/security/auth_policy_checker.h"
 
 #define CONFIG_PREFIX "/usr/share/stratum/dpdk/"
+#define LOG_DIR "/var/log/stratum/"
 
 DEFINE_string(dpdk_sde_install, "/usr",
               "Absolute path to the directory where the SDE is installed");
-DEFINE_bool(dpdk_infrap4d_background, false,
+DEFINE_bool(dpdk_infrap4d_background, true,
```

Comment:
Do we want to do this in all cases, or only when krnlmon is enabled?

You might want to add a comment explaining why we default to background mode for DPDK.

Reply:
We set the defaults before we parse the command line flags. The user still has the ability to override them.

Interesting thought, but I think I'd call it `dpdk_shell_background` rather than `bf_shell_background`. The `bf_` prefix is specific to the barefoot platform, and we use `dpdk_` for the other switches specific to the dpdk target.